### DISCUSSION TITLE
Translation - Default configuration

### DISCUSSION THREAD

**Discussion by Loupimo** [Loupimo/UniversalSwatchSlots#7] [Ideas]
Please post your JSON config file containing the default palette translated in the language you want.
Name it USS-XXX. where XXX is the language acronym (e.g. USS-GER for the german translation)
Big thanks in advance ❤️ 

They will be integrated in the next update and you will be credited (I when the custom config UI will be ready xD)

**Comment by krakel** [comment]
[USS-GER.json](https://github.com/user-attachments/files/17879400/USS-GER.json)

**Reply by Loupimo** [reply]
Thank you 😄

**Comment by krakel** [comment]
For fun :)
[USS-Minecraft.json](https://github.com/user-attachments/files/17879402/USS-Minecraft.json)

**Reply by Loupimo** [reply]
Nice one :)
As it is a custom palette could you please move it here : https://github.com/Loupimo/UniversalSwatchSlots/discussions/8
Thanks for sharing !

**Comment by CTRQuko** [comment]
[USS-SPANISH-SPAIN.json](https://github.com/user-attachments/files/18137721/USS-SPANISH-SPAIN.json)

**Reply by Loupimo** [reply]
Thanks, it will be added in the next update :)
